### ISSUE TITLE
LightHouse acsessibilty

### ISSUE THREAD

**Issue by TarasMykytiuk** [TarasMykytiuk/Project-Codewars-Leaderboard#2]
The assessment would have failed because lighthouse fails if a user occurred when fetching some data because the red on white doesn't have sufficient colour contrast. It's important to test your app for accessibility, including after dynamic changes happen to the UI.
